src/reanimated2/Hooks.js

```diff
@@ -417,7 +417,8 @@ function areDependenciesEqual(nextDeps, prevDeps) {
   var objectIs = typeof Object.is === 'function' ? Object.is : is;
 
   function areHookInputsEqual(nextDeps, prevDeps) {
-    if (!nextDeps || !prevDeps || (prevDeps.length !== nextDeps.length)) return !1;
+    if (!nextDeps || !prevDeps || prevDeps.length !== nextDeps.length)
+      return !1;
```

Comment:
Can we use {} brackets here and in `for, if` below? Also, could you change those `!1` & `!0`?

Reply:
d65eea1

Reply:
Thanks!